src/kubeobjects/platform.go

```diff
@@ -15,22 +20,54 @@ const (
 	Openshift
 )
 
-func ResolvePlatformFromEnv() Platform {
-	switch os.Getenv(platformEnvName) {
-	case openshiftPlatformEnvValue:
-		return Openshift
-	case kubernetesPlatformEnvValue:
-		fallthrough
-	default:
-		return Kubernetes
+const SccGVR = "security.openshift.io/v1"
```

Comment:
😭 🤮  (I don't have a better name, `OpenshiftSecurityGVR`, but its not much better)